### PR TITLE
[Feature] Add tracking docs

### PR DESCRIPTION
**Pull request by zwhus**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/user_guides/mot/4_train_test.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+```shell script
+bash ./tools/slurm_test_tracking.sh ${PARTITION} ${JOB_NAME} ${CONFIG_FILE} ${GPUS}
+```
+
+An example of testing the MOT model QDTrack with Slurm:
+
+```shell script
+PORT=29501 \
+GPUS_PER_NODE=8 \
+SRUN_ARGS="--quotatype=reserved" \
```
Comment (JingweiZhang12) [inline docs/en/user_guides/mot/4_train_test.md:218]: This  `GPUS_PER_NODE` and `quotatype` can be removed.

File: docs/en/user_guides/mot/5_visualization.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- `name`: Name of the instance. Defaults to 'visualizer'.
+- `image`: The origin image to draw. The format should be RGB. Defaults to None.
+- `vis_backends`: Visual backend config list. Defaults to None.
+- `save_dir`: Save file dir for all storage backends. If it is None, the backend storage will not save any data.
+- `bbox_color`: Color of bbox lines. The tuple of color should be in BGR order. Defaults to None.
+- `text_color`: Color of texts. The tuple of color should be in BGR order. Defaults to (200, 200, 200).
+- `line_width`: The linewidth of lines. Defaults to 3.
+- `alpha`: The transparency of bboxes or mask. Defaults to 0.8.
+
+Here is a visualization example of DFF:
```
Comment (JingweiZhang12) [inline docs/en/user_guides/mot/5_visualization.md:47]: remove DFF and add a new demo image link.

File: docs/en/user_guides/mot/1_config.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- `val_dataloader`: the config of validation dataloader, which is similar with `train_dataloader`.
+- `test_dataloader`: the config of testing dataloader, which is similar with `train_dataloader`.
+- `val_evaluator`: the config of validation evaluator. For example,`type='MOTChallengeMetrics'` for MOT task on the MOTChallenge benchmarks.
+- `test_evaluator`: the config of testing evaluator, which is similar with `val_evaluator`.
+- `train_cfg`: the config of training loop. For example, `type='EpochBasedTrainLoop'`.
+- `val_cfg`: the config of validation loop. For example, `type='VideoValLoop'`.
+- `test_cfg`: the config of testing loop. For example, `type='VideoTestLoop'`.
+- `default_hooks`: the config of default hooks, which may include hooks for timer, logger, param_scheduler, checkpoint, sampler_seed, visualization, etc.
+- `vis_backends`: the config of visualization backends, which uses `type='LocalVisBackend'` as default.
+- `visualizer`: the config of visualizer. For example, `type='DetLocalVisualizer'` for VID task, and `type='TrackLocalVisualizer'` for MOT, VIS, SOT, VOS tasks.
```
Comment (JingweiZhang12) [inline docs/en/user_guides/mot/1_config.md:25]: No VID

File: docs/en/user_guides/mot/1_config.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+- **Update values of list/tuples.**
+
+  Maybe the value to be updated is a list or a tuple.
+  For example, you can change the key `mean` of `data_preprocessor` by specifying `--cfg-options model.data_preprocessor.mean=[0,0,0]`.
+  Note that **NO** white space is allowed inside the specified value.
+
+## Config File Structure
+
+There are 3 basic component types under `config/_base_`, i.e., dataset, model and default_runtime.
+Many methods could be easily constructed with one of each like DFF, FGFA, SELSA, SORT, DeepSORT.
```
Comment (JingweiZhang12) [inline docs/en/user_guides/mot/1_config.md:59]: This line

File: docs/en/user_guides/mot/2_dataset_prepare.md [hunk 5]
```diff
@@ -0,0 +1,385 @@
...
+
+#### 1.2 Video Instance Segmentation
+
+- For the training and testing of video instance segmetatioon task, only one of YouTube-VIS datasets (e.g. YouTube-VIS 2019) is needed.
+
+- YouTube-VIS 2019 dataset can be download from [OpenDataLab](https://opendatalab.com/) (recommended for users in China): https://opendatalab.com/YouTubeVIS2019/download
+
+#### 1.5 Data Structure
+
+If your folder structure is different from the following, you may need to change the corresponding paths in config files.
```
Comment (JingweiZhang12) [inline docs/en/user_guides/mot/2_dataset_prepare.md:41]: The datasets we have not supported in stage-1 could be removed.

File: docs/en/user_guides/tracking_analysis_tools.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
+**We provide lots of useful tools under the `tools/` directory.**
```
Comment (RangiLyu) [inline docs/en/user_guides/tracking_analysis_tools.md:1]: You need to add these docs to the index.rst
